fastestimator/op/numpyop/multivariate/affine.py

```diff
@@ -74,7 +76,8 @@ def __init__(self,
                  border_handling: Union[str, List[str]] = "reflect",
                  fill_value: Number = 0,
                  interpolation: str = "bilinear",
-                 mode: Optional[str] = None,
+                 mode: Union[None, str, Iterable[str]] = None,
+                 ds_id: Union[None, str, Iterable[str]] = None,
```

Comment:
Optional Type Hint

Reply:
Optional is unnecessary / undesirable when dealing with more than one real option in a Union